tests/ping.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package tests
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	expect "github.com/google/goexpect"
+	"google.golang.org/grpc/codes"
+	"k8s.io/utils/net"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+var (
+	promptPriv   = regexp.MustCompile("\\# ")
+	promptUnPriv = regexp.MustCompile("\\$ ")
+	shellSuccess = regexp.MustCompile("\n0")
```

Comment:
The `shellSuccess` should contain the prompt as well. So the prompt won't be left as a left over in the buffer ([this](https://github.com/kubevirt/kubevirt/pull/3882/commits/7d3b5ea1d4964ba9fdfb83f305d5c2399cb28c68) commit changes `tests.Retcode` to contain the prompt, if it is merged before this PR it can be used).

Reply:
This `\n0` appears post the prompt, so I do not expect it to be left in the buffer.

We also do not know the prompt when the ping is called, and I do not want to know it.

Reply:
I think I got it, I will take care of this although IMO it is not needed in this case because the expecter is closed and no buffer is expected to be left.